bin/comBGC.py

```diff
@@ -131,7 +141,9 @@
 
 # Make sure that at least one input argument is given
 if not (input_antismash or input_gecco or input_deepbgc or dir_antismash):
-    exit("Please specify at least one input file (i.e. output from antismash, deepbgc, or gecco) or see --help")
+    exit(
```

Comment:
You mean with try/raise/except/finally...? I'll have to look into that for next version. Because I wanted a clear/informative error message for the user (not a traceback message if possible).